build-system/tasks/serve.js

```diff
@@ -191,6 +193,7 @@ module.exports = {
   doServe,
   startServer,
   stopServer,
+  DEFAULT_PORT,
```

Comment:
I have a related-ish PR in review that also needs to make use of the host/port used by the server: https://github.com/ampproject/amphtml/pull/30129/files#diff-68eb307bd570cd1da4673aa2f01e235c
It makes similar changes with setting the default at the top, but does so for both the port and the host.

Instead of exporting `DEFAULT_PORT`, WDYT about exporting `HOST` and `PORT` as above? Doing it as above means your consuming code below can directly get the port in use by the server without having to also check `argv`